cdc/kv/client.go

```diff
@@ -232,7 +242,12 @@ func NewCDCClient(
 	return c
 }
 
-func (c *CDCClient) newStream(ctx context.Context, addr string, storeID uint64) (stream *eventFeedStream, newStreamErr error) {
+func (c *CDCClient) newStream(
+	ctx context.Context,
+	cancel context.CancelFunc,
```

Comment:
pass the `cancel` as a function argument looks so weired.

Reply:
I have removed it.